File: Vishwas RNO 40/DotNet/Assignment 5/DemoApp/finance/Loan.cs
namespace finance;

public abstract class Loan
{
    public double principle{ get; set; }

    public double period { get; set;}

    public abstract double GetRate();

    public double EMI()
    {
        double EMI = principle * ( 1 + GetRate() * period / 100 ) / ( 12 * period );      
        return EMI;
    }
}

File: Vishwas RNO 40/Java/Assign6/src/Loan.java
package finance;
public abstract class Loan {
	
	private double principle;
	private float period;

	public abstract float getRate();

	public double EMI(){	
		
		double EMI = principle * (1 + getRate() * period / 100 ) / (12 * period);
		return EMI;
	}
	public double getPrinciple() {
		return principle;
	}

	public void setPrinciple(double principle){
		this.principle = principle;
	}

	public float getPeriod() {
		return period;
	}

	public void setPeriod(float period) {
		this.period = period;
	}

}
